doc/setup.md

```diff
@@ -11,3 +11,32 @@
 
 1. A Nitro Enclave requires a minimum of 2 vCPUs and a host can have at most 4 Enclaves, more information [here](https://docs.aws.amazon.com/enclaves/latest/user/nitro-enclave.html)
 2. The host must always have at least 2 vCPUs, so the host machine must have at least 4 vCPUs to support 1 Enclave.
+
+
+# Installing dependencies 
+
+1. Install Docker
+```bash
+sudo yum -y install docker
+sudo usermod -a -G docker ec2-user
+sudo systemctl enable docker.service
+sudo systemctl start docker.service
+```
+
+2. Install Nitro CLI
+```bash
+sudo amazon-linux-extras install aws-nitro-enclaves-cli -y
+sudo yum install aws-nitro-enclaves-cli-devel -y
```

Comment:
On AMZ 2023 this is just `sudo yum install -y aws-nitro-enclaves-cli aws-nitro-enclaves-cli-devel`. Does that work on recent AMZ 2?

Reply:
I tried it, it doesn't work. This is what I had referenced earlier: https://docs.aws.amazon.com/enclaves/latest/user/nitro-enclave-cli-install.html

Reply:
Ok, thanks for checking.